test/test_data.h

```diff
@@ -72,129 +72,76 @@ ThingSetBytesBuffer bytes_buf = { bytes, 0 };
-    // using IDs >= 0xA0
+    // RECORDED DATA //////////////////////////////////////////////////////////
 
     TS_NODE_PATH(ID_REC, "rec", 0, NULL),
 
     TS_NODE_FLOAT(0xA1, "BatHour_kWh", &bat_energy_hour, 2, ID_REC, TS_ANY_R, 0),
     TS_NODE_FLOAT(0xA2, "BatDay_kWh", &bat_energy_day, 2, ID_REC, TS_ANY_R, 0),
     TS_NODE_INT16(0xA3, "AmbientMaxDay_degC", &ambient_temp_max_day, ID_REC, TS_ANY_R, 0),
 
-    // CALIBRATION DATA ///////////////////////////////////////////////////////
-    // using IDs >= 0xD0
+    // REMOTE PROCEDURE CALLS /////////////////////////////////////////////////
 
-    TS_NODE_PATH(ID_CAL, "cal", 0, NULL),
+    TS_NODE_PATH(ID_RPC, "rpc", 0, NULL),
 
-    // FUNCTION CALLS (EXEC) //////////////////////////////////////////////////
-    // using IDs >= 0xE0
-
-    TS_NODE_PATH(ID_EXEC, "exec", 0, NULL),
+    TS_NODE_EXEC(0xE1, "x-reset", &reset_function, ID_RPC, TS_ANY_RW),
+    TS_NODE_EXEC(0xE2, "x-auth", &auth_function, ID_RPC, TS_ANY_RW),
+    TS_NODE_STRING(0xE3, "Password", auth_password, sizeof(auth_password), 0xE2, TS_ANY_RW, 0),
 
-    TS_NODE_EXEC(0xE1, "reset", &reset_function, ID_EXEC, TS_ANY_RW),
+    // REPORTS ////////////////////////////////////////////////////////////////
 
-    TS_NODE_EXEC(0xE2, "auth", &auth_function, 0, TS_ANY_RW),
-    TS_NODE_STRING(0xE3, "Password", auth_password, sizeof(auth_password), 0xE2, TS_ANY_RW, 0),
+    TS_NODE_PUBSUB(0xF4, "report", PUB_REPORT, 0, TS_ANY_RW, 0),
 
     // PUBLICATION DATA ///////////////////////////////////////////////////////
-    // using IDs >= 0xF0
-
-    TS_NODE_PATH(ID_PUB, "pub", 0, NULL),
-
-    TS_NODE_PATH(0xF1, "serial", ID_PUB, NULL),
-    TS_NODE_BOOL(0xF2, "Enable", &pub_serial_enable, 0xF1, TS_ANY_RW, 0),
-    TS_NODE_UINT16(0xF3, "Interval_ms", &pub_serial_interval, 0xF1, TS_ANY_RW, 0),
-    TS_NODE_PUBSUB(0xF4, "IDs", PUB_SER, 0xF1, TS_ANY_RW, 0),
-
-    TS_NODE_PATH(0xF5, "can", ID_PUB, NULL),
-    TS_NODE_BOOL(0xF6, "Enable", &pub_can_enable, 0xF5, TS_ANY_RW, 0),
-    TS_NODE_UINT16(0xF7, "Interval_ms", &pub_can_interval, 0xF5, TS_ANY_RW, 0),
-    TS_NODE_PUBSUB(0xF8, "IDs", PUB_CAN, 0xF5, TS_ANY_RW, 0),
-
-    // LOGGING DATA ///////////////////////////////////////////////////////
-    // using IDs >= 0x100
-
-    TS_NODE_PATH(0x100, "log", 0, NULL),
-
-    TS_NODE_PATH(0x110, "hourly", 0x100, NULL),
-    /*
-        "hourly": {
-            "0h": {"Timestamp_s":23094834,"BatHour_kWh":123},
-            "-1h": {"Timestamp_s":23094834,"BatHour_kWh":123}
-        }
-    */
-
-    TS_NODE_PATH(0x130, "daily", 0x100, NULL),
-    /*
-        "daily": {
-            "0d": {"Timestamp_s":23094834,"BatDay_kWh":123,"AmbientMaxDay_degC":26},
-            "-1d": {"Timestamp_s":34209348,"BatDay_kWh":151,"AmbientMaxDay_degC":28}
-        },
-    */
-
-    // UNIT TEST DATA ///////////////////////////////////////////////////////
+
+    TS_NODE_PATH(ID_PUB, ".pub", 0, NULL),
```

Comment:
Is the dot intended?

Reply:
Yep, it is. The dot prefix is now used for internal data objects in spec v0.4.